src/engine/enginebuffer.cpp

```diff
@@ -260,7 +260,7 @@ EngineBuffer::EngineBuffer(const QString& group,
                                                m_pLoopingControl);
     m_pReadAheadManager->addRateControl(m_pRateControl);
 
-    m_pKeylockEngine = new ControlProxy("[Master]", "keylock_engine", this);
+    m_pKeylockEngine = new ControlProxy(kAppGroup, QStringLiteral("keylock_engine"), this);
```

Comment:
unrelated to the change, but we might want to consider switching `ConfigKey` from `QString` to `QLatin1String` as those can be significantly faster apparently and I don't think we need/want Unicode support in COs, do we?

Reply:
This makes sense, but I'd rather not do this here, because it would mean we need to change *a lot* of COs. This can go into `main` IMHO.

Reply:
Yes, I agree. Just wanted to raise the point so I don't forget about it.